implementations/code_df/reviews_declined_github.py

```diff
@@ -0,0 +1,70 @@
+from datetime import datetime
+
+import utils
+from pullrequest_github import PullRequestGitHub
+
+
+class ReviewsDeclinedGitHub(PullRequestGitHub):
+    """
+    Class for Reviews Declined
+    """
+
+    def compute(self):
+        """
+        Compute the total number of reviews which were declined, from the
+        Perceval data. Declined reviews are in the "closed" state but
+        are not merged.
+
+        :returns count: The total number of reviews declined
+        """
+
+        count = len(self.df['hash'][~self.df['merged']
```

Comment:
Why do you need `unique`? Sorry if it is obvious, but I don't get it...

Reply:
I don't think its required. I'm trying to think of a case when there would be multiple prs with the same id number, similar to how commits are the same in different forks.

Reply:
I'm removing it for now.